src/main/java/se/sundsvall/snailmail/api/SnailMailResource.java

```diff
@@ -19,12 +23,13 @@
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import io.swagger.v3.oas.annotations.tags.Tag;
-import jakarta.validation.Valid;
 
 @RestController
 @Validated
 @RequestMapping("/send/snailmail")
 @Tag(name = "SnailMailSender", description = "SnailMailSender")
+@ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(schema = @Schema(implementation = Problem.class)))
```

Comment:
Change to `schema = @Schema(oneOf = { Problem.class, ConstraintViolation.class })`?